### ISSUE TITLE
Figures - needed by Feb. 20th

### ISSUE THREAD

**Issue by emily-phet** [phetsims/tasks#916]
Figures:
- Figure 1: Screenshot of BASE on sim load
- Figure 2: Screenshot of RIAW on sim load
- Figure 3: Screenshot of JT on sim load - no pink boxes on arm/leg
- Figure 4: Two Panels - Left Panel: Only white slider control panel from RIAW sim, showing sliders in  a position different from the start, all numerical values not rounded (more difficult to work with). Right Panel: Only white slider control panel from RIAW sim, showing sliders in a position with either double or halve the L and A of the left panel,  all numerical values nice round numbers.
- Figure 5: Two Panels - Top Panel: JT sim, with overlay of 'pie' showing early set of description regions (similar right side of design mockup below). Bottom Panel: JT sim, with overlay of 'pie' showing current description regions, labeled with descriptions, and landmarks. Consult with @terracoda and @jessegreenberg to make sure representation matches current implementation.

![john-travoltage-3-08](https://user-images.githubusercontent.com/5109055/36062238-3c4349ba-0e25-11e8-8045-3054b2d2fb0b.png)

- Figure 6: 4 panels, looks like image below, but created to match look and resolution of previous figures.
[BaseImage.pdf](https://github.com/phetsims/tasks/files/1713374/BaseImage.pdf)

These figures will be used in a manuscript, with max width of about 4 inches.

**Comment by ariel-phet** [comment]
@amanda-phet said that @CWilson4960 should be able to do these screenshots.

The other figure probably requires some illustrator work and would be good practice for @DianaTavares

**Comment by amanda-phet** [comment]
@CWilson4960 do you think you can get to these by Wednesday 2/14?

**Comment by amanda-phet** [comment]
@phet-steele can do these in 5 minutes so he is the best person to do them right now. 

@emily-phet I think we need some clarification on the aspect ratio you want for screenshots, as well as some of the figures you are describing. Do you have time tomorrow to zoom about this?

**Comment by emily-phet** [comment]
@amanda-phet I don't have a strong feeling regarding aspect ratio. The manuscript is single column, with really wide margins, so the 'usable' space for text/figures is a little over 4 inches wide. The most important thing is that the screenshots all have the same aspect ratio.

Unfortunately, I don't have much time today to spare. Can you summarize the questions here, and I'll take a look and see if I can fit in an appropriate amount of time to discuss this afternoon or tomorrow.

**Comment by amanda-phet** [comment]
(1) Figure 4: This is  a screenshot of the panel from the sim, correct? At first I was reading this thinking it would be a slider like the one in your example, but that's probably why I was confused. 

(2) Figure 5: Do you want the entire JT sim in the screenshot, or just a cropped version like the one you showed as an example? Do you want the pie to look like the one in your example? Do you already have an illustrator file with that pie that we can re-use?  

(3) Figure 6: This is simply 4 screenshots, arranged in a grid, correct? Would you still like the drop shadow or no shadow? 

(4) How would you like these figures saved? Is there a particular resolution you want? File format?

**Comment by emily-phet** [comment]
@amanda-phet 

(1) Figure 4: This is a screenshot of the panel from the sim, correct? At first I was reading this thinking it would be a slider like the one in your example, but that's probably why I was confused.
- yes, I'm referring to the white panel that holds all of the sim's sliders.

(2) Figure 5: Do you want the entire JT sim in the screenshot, or just a cropped version like the one you showed as an example? Do you want the pie to look like the one in your example? Do you already have an illustrator file with that pie that we can re-use?
- A cropped image of JT is fine if that's easier to work with. I don't have a preference either way.  Unfortunately, I don't have the original files used to make the image example - this was created by IDRC folks a long time ago. The "pie" doesn't have to look exactly like the example image. In the top panel of the two-panel image, the pie will have the same number of regions as the example figure (essentially, the top panel image needs to reproduce the content of the example image, though it doesn't have to look exactly the same). The bottom panel will have a pie representing the actual implemention of the published JT - which has 30 total regions. Please consult with @terracoda on the details. This will likely be the most challenging to create, as it's new and we'll need to figure out the best way to convey a relatively dense amount of information.

(3) Figure 6: This is simply 4 screenshots, arranged in a grid, correct? Would you still like the drop shadow or no shadow?
- Correct, four screenshots arranged in a grid. My preference is that the screenshots look consistent with the others in the manuscript, so whatever the previous figures have is fine.

(4) How would you like these figures saved? Is there a particular resolution you want? File format?
- I don't have a particular resolution in mind. The images just need to be crisp at about a 4-inch max height. I need to embed the images into a word file, and then export as a pdf. So anything I can drop into a Word file will work. If you could save the final version on Google Drive in the folder [here](https://drive.google.com/drive/folders/17vDCWhBtwLY3gTJ5B7_W8zenVE7SzBKG?usp=sharing) that would be helpful.

(5) In figure 3 you specify no pink boxes.. does that mean you want them in figures 1 and 2? [edit: Steele and I realized you meant the startup boxes that are special to JT. They are also green in the version I was looking at, which threw us off]
- Sorry for being unclear - we have so many pink visual cues now!

**Comment by phet-steele** [comment]
@amanda-phet @emily-phet @DianaTavares I believe the screenshots needed from me are done and placed here: https://drive.google.com/drive/folders/17vDCWhBtwLY3gTJ5B7_W8zenVE7SzBKG?usp=sharing

Let me know if anything else is needed. Have fun!.

**Comment by terracoda** [comment]
@amanda-phet and @DianaTavares 

Rough arrangement for 60 positions was somewhat like this, but I see my ranges here are more even than the the one in Jon Hung's diagram:

"Position 0, farthest from the door knob".

"Position 1 to 4, very far from the door knob".
"Position 5 to 9, far from the door knob".
“Position 10 to 14, neither far or close to the door knob”.
"Position 15 to 19, somewhat close to the door knob".
"Position 20 to 24, close to the door knob".

"Position 25 to 29, very close to the door knob".

"Position 30, closest to the door knob".

"Position 31 to 35, very close to the door knob".

"Position 36 to 40, close to the door knob".
"Position 41 to 45, somewhat close to the door knob".
“Position 46 to 50, neither far or close to the door knob”.
"Position 51 to 55, far from the door knob".
"Position 56 to 59, very far from the door knob".

"Position 60, farthest from the door knob".

**Comment by DianaTavares** [comment]
Hi @terracoda, this is the Figure. The .ai file is in Drive

![screen shot 2018-02-20 at 12 45 13 pm](https://user-images.githubusercontent.com/26031501/36445646-1c5b52f4-163c-11e8-900b-4cbc5baaf021.png)

What are the labels for the second pieJT Figure?  The description said that the pie have to show the "current description regions"

**Comment by terracoda** [comment]
@amanda-phet and @DianaTavares , I think the circle needs to have a gap at the end.
I will draw some pictures and post asap. Likely tomorrow morning.

**Comment by terracoda** [comment]
@DianaTavares, Looking great... just finished, my pencil drawing for the comparison. It is king of complicated and hard to see. Will take a closer shot, since you have done the upper one already.

**Comment by terracoda** [comment]
@DianaTavares, could you put a gap in the red region to show that the it start at 0 and ends at 60?

**Comment by terracoda** [comment]
@DianaTavares, I will add the text in the next comment
![figure-arm-comparison](https://user-images.githubusercontent.com/1902116/36495529-1eb8764c-1710-11e8-902e-27c790667b03.jpg)

**Comment by DianaTavares** [comment]
I see that this pie has 9 section + 1 little section. And one section (the one closer to the door) is divided in other 4 sections, and the section in the bottom is divided in two. I am going to start with the pie for now. 

 I don't see very well the text. Do I use the same color that the first pie?

**Comment by terracoda** [comment]
@DianaTavares, it still has 12 regions...I am typing out text. Will post in a minute.

**Comment by DianaTavares** [comment]
ok! first I made the  gap at the end of the first pie picture. I am no sure if you mean something like this. Because other option is to draw a black or red line between position 0 and 60. What do you like more? 

![screen shot 2018-02-21 at 11 02 21 am](https://user-images.githubusercontent.com/26031501/36496885-e7f9cb88-16f6-11e8-9b2b-e6fe32b528c4.png)

**Comment by terracoda** [comment]
Here's the text:
Postion -15: Farthest from doorknob. Last stop. [Type Landmark]

Position -14 to -12: Very far from doorknob. [Type Region]
 - Position -13: Hand poiting away from door, very far from doorknob. [Prefix adds orientation]

Position -11 to -9: Far from doorknob. [Type Region]

Position -8: Hand pointing straight up. [Type Landmark]

Position -7 to -6: Not so close to doorknob. [Type Region]

Position -5 to -3: Close to doorknob.
- Position -4: Hand pointing at upper door, close to doorknob. [Prefix adds orientation]

Position -2: Very close to doorknb.

Position -1: Just above doorknob.

Position 0: At doorknob. [Type Landmark]

Position 1: Just below doorknob.

Position 2: Very close to doorknb.

Position 3 to 5: Close to doorknob.
- Position 4: Hand pointing at lower door, close to doorknob. [Prefix adds orientation]

Position 6: Not so close to doorknob. [Type Region]
Position 7: Hand pointing straight up. [Type Landmark]
Position 8: Not so close to doorknob. [Type Region]

Position 9 to 11: Far from doorknob. [Type Region]

Position 12 to 14: Very far from doorknob. [Type Region]

Position 13: Hand poiting away from door, very far from doorknob. [Type Region, but with orientation]

Position 15: Farthest from doorknob. Last stop. [Type Landmark]

**Special note:**
For orintation positions -2 and 2 change when user is moving away
to "Farther away, very close to doorknob."

**Comment by DianaTavares** [comment]
this one has a better the gap at the end, because the red sections have the same side and the gap don't have a black line from the circle. Looks more like the end of a cicle. 
![screen shot 2018-02-21 at 11 10 52 am](https://user-images.githubusercontent.com/26031501/36497283-19307eee-16f8-11e8-8b85-33197c5fdb34.png)

I start to work in the other picture

**Comment by DianaTavares** [comment]
@terracoda 

I made the divisions of the circle using your drawing. Three questions:
-I made the division as they are in your drawn. Do you want them that way or you want it all the same size?
-What do you think about the colors? 
-I understand that you want some text inside the triangles, and some outside(o,1,-1 for example). But I don't understand if you want some text 6 and 8 and the full text for the 13 and -13:
"Hand poiting away from door, very far from doorknob"

![screen shot 2018-02-21 at 12 05 17 pm](https://user-images.githubusercontent.com/26031501/36500425-7db8f90a-1701-11e8-90b1-26662453b2cd.png)

**Comment by amanda-phet** [comment]
Another question for @terracoda : 

Do you want position 30 (top panel) and position 0 (bottom panel) to be in the 3-o'clock position? Or would you like it rotated so that it is actually closest to the doorknob? 

This is important to know because it will make the diagrams easier to compare if the decision is the same for both, and it impacts how @DianaTavares draws the segments in the circle because position -8 and 7 need to be at 12-o'clock and 6 o'clock to represent straight up and straight down.

**Comment by terracoda** [comment]
Rotated so the green stripe is "At the doorknob."

**Comment by terracoda** [comment]
The -8 and 7 are not perfect visually, but that's ok.
See the position numbers in this version of the John Travoltage:
https://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-dev.36/john-travoltage-a11y-view.html?valueText

**Comment by terracoda** [comment]
What is imporant to convey comparison is the following:
- reduction in number of positions from 61 to 31
- the new range orientation from -15 to 15 with 0 at the doorknob.
- the addition of landmarks and positions with orientation. I think logically these behave the same way.
  - Negative 8 and Positive 7 for straight up and straight down.
  - Negative 4 and Positive 4 for Pointing to upper and lower door.
  - Negative 13 and Positive 13 for Pointing away from door.
  - Negative 15 and Positive 15 are Last stops.
  - Position zero is the doorknob.

There are other dynamic orientational description for positions Negative 2 and Positive 2, and for changes in direction, but I do not think that can be conveyed on figure.

**Comment by terracoda** [comment]
Just going to throw this out there. From a designer's perspective, do think this would better compared with side by side vertical representations of the slider and slider content, and one clean picture of the pie on top of John's arm?

**Comment by terracoda** [comment]
This is totally rough. Do you understand it?

![rough-side-by-side](https://user-images.githubusercontent.com/1902116/36502377-50385d0e-1724-11e8-86b8-dee48d97726d.jpg)

**Comment by terracoda** [comment]
@DianaTavares, it might be way easier to display the text vertically, and it might actually communicate the comparison more efficiently.

What do you think?

**Comment by DianaTavares** [comment]
for now I have this:
![screen shot 2018-02-21 at 1 10 29 pm](https://user-images.githubusercontent.com/26031501/36502920-a3c5e0e8-1708-11e8-97fc-fbf6dde026a0.png)

I rotate both pies, and now the green line match with the closer position from the doorknob. 
But I don't see them like similares to compare. But put all the text around the second circle is not posible because it's a lot of text, and some text is in the same spot (like -15 and -14)

I think that vertically can be easier

**Comment by DianaTavares** [comment]
last version
![screen shot 2018-02-21 at 4 27 06 pm](https://user-images.githubusercontent.com/26031501/36511502-22f32838-1724-11e8-9528-a909ed61587b.png)

**Comment by terracoda** [comment]
@DianaTavares, I think the last one is the best.

**Text Changes**
- Punctuation should be the same on both sides, e.g use colons on both sides and capitalize first word. (**sorry I didn't make that clear in the text I provided**)
- Fix **my typos** in -13 poiting -> pointing

**Design changes**
Any chance we can add some line-height to make things less crowded?
Add a title row with column headers:
- Table 1 header row cells:
  - Position #  
  - Position descriptions
- Table 2 header row cells:
  - Position #  
  - **Main Landmark** and Region Descriptions
- Stretch color column right do it is as wide as the words in the header cell "Position" 
- Remove all instances of "Position" in the table rows, and use only actual numbers. That might make things less busy looking.

**Comment by terracoda** [comment]
@emily-phet, @DianaTavares 

I updated the illustrator file and uploaded new versions to the google drive folder and I created a hi-res JPG (also uploaded to the folder).

Waiting to hear back from @emily-phet to see if the JPG file looks nice when added to the manuscript.

For visual reference here the JPG file:
![figure5_jt-arm-comparison](https://user-images.githubusercontent.com/1902116/36604978-e3e0b9a4-1899-11e8-995e-7254d62fe6aa.jpg)

**Comment by terracoda** [comment]
@emily-phet and @DianaTavares, in addition, I didn't see a way to make a PDF in illustrator of the figure which is why I created a hi-res JPG.

**Comment by terracoda** [comment]
For documentation purposes, @emily-phet  and @terracoda decided to remove the circles and just go with the tables.

**Comment by terracoda** [comment]
@emily-phet, @DianaTavares found some inconsistencies in font size that I introduced, and has cleaned that up. I will send you a new image.

**Comment by terracoda** [comment]
@emily-phet, @DianaTavares fixed the font sizing.
I uploaded a new hi-res JPG file for figure 5 to the folder. where the font size is all the same size.

![figure5_jt-arm-comparison](https://user-images.githubusercontent.com/1902116/36610885-459271da-18ac-11e8-8457-f8e3d051e0d3.jpg)

**Comment by terracoda** [comment]
Image has been approved. Closing the issue.
